### PR TITLE
Feature/weekly reset

### PR DESCRIPTION
**Pull request by ajg7**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: api/cronTasks/cronTasks.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                            .groupBy('Children.ID')
+
+    for (const child of derivedTable) {
+        const ID = child.ID;
+        const sum = child.sum;
+        db('Children').where({ ID }).update({ Total_Points: Total_Points + sum })
+    }
+}
+
+// update "Children" as "c"
```
Comment (DeeDowns) [inline api/cronTasks/cronTasks.js:19]: is this code that will be implemented later?

File: api/profile/profileRouter.js [hunk 2]
```diff
@@ -100,7 +98,7 @@ const { ops } = require('../../lib');
 router.get('/', authRequired, async (req, res) => {
   const { Email } = req.profile;
 
-  ops.getAll(res, Parents.getProfilesByEmail, 'Profile', Email);
+  crudOperationsManager.getAll(res, Parents.getProfilesByEmail, 'Profile', Email);
```
Comment (DeeDowns) [inline api/profile/profileRouter.js:101]: is` crudOperationsMangager` doing the same thing as the `ops` object? you just replaced it and it has extra functionality for the weekly reset?

**Review by DeeDowns — CHANGES_REQUESTED** [review]
overall looks good, just the bit about crudOperationsManager replacing ops, I'm sure theres a reason for the change but its not mentioned. Could you explain what the difference is before I approve?

**Review by DeeDowns — APPROVED** [review]
Ok thank you for explaining the semantic reasoning behind the crudOperationsManager changes. Looks good

**Review by gregory-j-wilson — APPROVED** [review]
looks good

**Pull request merged**